pkg/config/draftconfig_test.go

```diff
@@ -0,0 +1,204 @@
+package config
+
+import (
+	"testing"
+)
+
+func TestApplyDefaultVariables(t *testing.T) {
+	tests := []struct {
+		testName     string
+		draftConfig  DraftConfig
+		customInputs map[string]string
+		want         map[string]string
+		wantErr      bool
```

Comment:
I like the way you implemented this! I feel like checking the specific type of error you're getting back vs a bool to check if you got any error would be a bit more thorough though

Reply:
It is now checking for a specific error message!